Kiosk/Bid Fulfillment/KeypadContainerView.swift

```diff
@@ -4,39 +4,58 @@ import RxSwift
 import Action
 import FLKAutoLayout
 
-//@IBDesignable
+@IBDesignable
 class KeypadContainerView: UIView {
     fileprivate var keypad: KeypadView!
     fileprivate let viewModel = KeypadViewModel()
     
     var stringValue: Observable<String>!
     var currencyValue: Observable<Currency>!
     var deleteAction: CocoaAction!
+    var addPlusAction: CocoaAction!
     var resetAction: CocoaAction!
 
+    /// Setting this value after the instance has been loaded has no effect.
+    /// The value is set in Interface Builder.
+    @IBInspectable
+    var isPhoneNumberEntry: Bool = false
+
     override func prepareForInterfaceBuilder() {
-        for subview in subviews { subview.removeFromSuperview() }
+        super.prepareForInterfaceBuilder()
+
+        keypad = Bundle(for: type(of: self)).loadNibNamed("KeypadView", owner: self, options: nil)?.first as? KeypadView
 
-        let bundle = Bundle(for: type(of: self))
-        let image  = UIImage(named: "KeypadViewPreviewIB", in: bundle, compatibleWith: self.traitCollection)
-        let imageView = UIImageView(frame: self.bounds)
-        imageView.image = image
+        if self.isPhoneNumberEntry {
+            keypad.setLeftButton(.plus)
+        } else {
+            keypad.setLeftButton(.delete)
+        }
```

Comment:
Is there a delete/clear button for non-phone numbers?

Reply:
That's right, we still use this UI for entering bid amounts.